syntax = "proto3";

option java_multiple_files = true;
option java_package = "org.e8yes.service";
option java_outer_classname = "PaginationProto";

message Pagination {
    int32 page_number = 3;
    int32 result_per_page = 4;
}
